zellij-utils/src/data.rs

```diff
@@ -251,6 +251,35 @@ impl FromStr for Direction {
     }
 }
 
+#[derive(Eq, Clone, Copy, Debug, PartialEq, Hash, Deserialize, Serialize, PartialOrd, Ord)]
+pub enum HorizontalDirection {
```

Comment:
Can't we re-use an already existing `Direction` enum?

When the actual API is called, the `ProtobufMoveTabDirection` data appears to be mapped to a `HorizontalDirection`, so it looks like the existing `Direction` can be used. (If you handle an exception in the vertical case)

What do you think?

Reply:
Yeah, I was thinking about that. I figured that there already is `SearchDirection`, `ResizeDirection`, `SpliDirection` etc., so I introduced `HorizontalDirection`, but I don't mind reusing `Direction`, it makes sense.

Please just make sure I handled the [vertical case exception](https://github.com/zellij-org/zellij/pull/3047/files#diff-6f77a56f2f7ebfaa89005876b659af0aa348b809cf0eebda1814e1c7454a637cR541) in a proper way, because I'm not entirely sure :thinking: